packages/core/src/modules/basic-messages/BasicMessagesModule.ts

```diff
@@ -48,15 +47,9 @@ export class BasicMessagesModule {
       connection
     )
     const outboundMessage = createOutboundMessage(connection, basicMessage)
-    try {
-      await this.messageSender.sendMessage(outboundMessage)
-    } catch (error) {
-      throw new MessageSendingError(`Error sending basic message: ${error.message}`, {
-        agentMessage: basicMessage,
-        associatedRecord: basicMessageRecord,
-        cause: error,
-      })
-    }
+    outboundMessage.associatedRecordId = basicMessageRecord.id
```

Comment:
Adding the id but not the type can make it hard to get the record type in a generic context. A possibility is attachign the whole record, but I like the id appraoch better.. What do you think?

Reply:
I was also hesitating on using the whole record or just the id. Opted to go for the id in order to make this whole outbound message more 'lightweight' and considering that the context would know the type of record to search for. But I don't have a very strong opinion on this. Can change to the whole record if we foresee that it's more convenient.